app/src/test/java/com/mybus/model/RoadResultUnitTest.java

```diff
@@ -0,0 +1,68 @@
+package com.mybus.model;
+
+import com.mybus.helper.FileLoaderHelper;
+import com.mybus.model.Road.RoadResult;
+
+import org.json.JSONObject;
+import org.junit.Test;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by ldimitroff on 12/05/16.
+ */
+public class RoadResultUnitTest {
+
+    /**
+     * Test a simple road result with type 0
+     */
+    @Test
+    public void parseSimpleRoadResult() {
+        RoadResult expectedResult = new RoadResult();
+        expectedResult.setType(0);
+        expectedResult.setTotalDistance(3F);
+        expectedResult.setTravelTime(12);
+
+        RoadResult roadResult = RoadResult.parse(FileLoaderHelper.loadJSONObjectFromResource(this, "road_result/road_result_simple.json"));
+        assertNotNull("Result is 'Null'", roadResult);
+
+        assertTrue(expectedResult.getType() == roadResult.getType());
```

Comment:
Would be good to implement equals and hashcode methods in RoadResult class and then replace the property-based assertions by this one;

```java
assertEquals(expectedResult, roadResult);
```